docs/source/aws.rst

```diff
@@ -0,0 +1,43 @@
+# How to ingest data from AWS
+
+Add AWS credentials to docker-compose
+
+Docker-compose has two repeated lines:
+
+AWS_ACCESS_KEY_ID=
+
+AWS_SECRET_ACCESS_KEY=
+
+You'll need to set those environment variables with your AWS access and secret key. And then you can call `docker-compose up`
+
+1)  Create a Provider
+
+Send a POST request to
+
+`http://<your_host>:8000/api/cost-management/v1/providers/`
+
+With the content:
+
+Provider Resource Name: <whatever your arn is>
```

Comment:
Might be worth it to have an example call here:
```
http POST 0.0.0.0:8000/api/cost-management/v1/providers/ name='AWSProvider' type='AWS' \
authentication:='{"uuid": "e6d90e5a-feb9-472e-97f3-b96839206f4f","provider_resource_name": "arn:aws:iam::01234567890:my_aws_role"}'
```

Reply:
@werwty I added this now!